skate-plugin/src/main/kotlin/com/slack/sgp/intellij/SkatePluginSettings.kt

```diff
@@ -26,7 +26,7 @@ import org.jetbrains.annotations.VisibleForTesting
 internal const val DEFAULT_TRANSLATOR_SOURCE_MODELS_PACKAGE_NAME = "slack.api.schemas"
 @VisibleForTesting internal const val DEFAULT_TRANSLATOR_FILE_NAME_SUFFIX = "Translator.kt"
 @VisibleForTesting internal const val DEFAULT_TRANSLATOR_ENUM_IDENTIFIER = "name"
-internal const val DEFAULT_CODE_OWNER_FILE_PATH = "config/code-ownership/code_ownership.csv"
+internal const val DEFAULT_CODE_OWNER_FILE_PATH = "code_health_team_config.yaml"
```

Comment:
Let's remove this default actually (default to missing) and check in the path directly in `skate.xml` in consuming projects

Reply:
makes sense, removed.